src/models/game/actions.js

```diff
@@ -8,8 +8,19 @@ const Action = (type) => {
 };
 
 export const check = Action("CHECK");
+export const gameChecked = Action("GAME_CHECKED");
+export const gameWin = Action("GAME_WIN");
+export const gameLose = Action("GAME_LOSE");
+export const gameTie = Action("GAME_TIE");
```

Comment:
GAME prefix seems irrelevant with the actual actions. Check also for game folded. They are more player related, aren't they?

Reply:
Done.